docs/test_plans/tp-001-devnet-full-node.md

```diff
@@ -0,0 +1,71 @@
+# Test Plan #001: Devnet Celestia Full Node basic communication
+
+## Introduction
+
+For Devnet stage, Celestia Full node can have several setup options to communicate with Celestia Core. The latter is a tendermint consensus node. For further text, we will refer to acronyms
+
+- Celestia Full Node = CFN
+- Celestia Core Node = CCN
+
+Setup options for CFN can be described with this list:
+
+1. `default` flag, which embeds CCN(acronym -> emCCN)
+2. `--core.disable` flag, which excludes CCN
+3. `--core.remote` flag, which connects to an external CCN(acronym -> rmCCN)
+
+## In-Scope
+
+Focus on CFN setup options
+
+- Integration Testing for CFN default/remote CCN setups
+- Subsystem Testing for communication of CFNs(emCCN) and CFNs(rmCCN)  
+
+## Out-of-Scope
+
+- CFN with disabled CCN (will be covered in #7)
+- Light clients
+- Performance testing scenario N->1 for CFN->CCN
+
```

Comment:
Other things out of scope: 
- Fraud proofs 
- Block time testing in Celestia Core 
- P2P layer propagation 
- Worst case scenario transaction time
@Bidon15

Reply:
Agreed. I will add it in the next commit